services/ui-src/src/forms/mcpar/ebssei/ebssei.json

```diff
@@ -1,18 +1,159 @@
+      }
+    ]
+  },
+  "drawer": {
+    "dashboard": {
+      "title": "Report on role and type for each BSS entity",
+      "entityType": "entities"
+    },
+    "drawerTitle": "Report on"
   },
   "form": {
     "id": "ebssei",
     "fields": [
       {
         "id": "ebssei-1",
-        "type": "text",
+        "type": "checkbox",
+        "props": {
+          "label": "E.IX.1 BSS entity type",
+          "hint": "What type of entity was contracted to perform each BSS activity? Check all that apply. Refer to 42 CFR 438.71(b).",
+          "choices": [
+            {
+              "name": "ebssei-1-o1",
+              "type": "choice",
+              "label": "State Government Entity"
+            },
+            {
+              "name": "ebssei-1-o2",
+              "type": "choice",
+              "label": "Local Government Entity"
+            },
+            {
+              "name": "ebssei-1-o3",
+              "type": "choice",
+              "label": "Ombudsman Program"
+            },
+            {
+              "name": "ebssei-1-o4",
+              "type": "choice",
+              "label": "State Health Insurance Assistance Program (SHIP)"
+            },
+            {
+              "name": "ebssei-1-o5",
+              "type": "choice",
+              "label": "Aging and Disability Resource Network (ADRN)"
+            },
+            {
+              "name": "ebssei-1-o6",
+              "type": "choice",
+              "label": "Center for Independent Living (CIL)"
+            },
+            {
+              "name": "ebssei-1-o7",
+              "type": "choice",
+              "label": "Legal Assistance Organization"
+            },
+            {
+              "name": "ebssei-1-o8",
+              "type": "choice",
+              "label": "Other Community-Based Organization"
+            },
+            {
+              "name": "ebssei-1-o9",
+              "type": "choice",
+              "label": "Subcontractor"
+            },
+            {
+              "name": "ebssei-1-o10",
+              "type": "choice",
+              "label": "Enrollment Broker"
+            },
+            {
+              "name": "ebssei-1-o11",
+              "type": "choice",
+              "label": "Consultant"
+            },
+            {
+              "name": "ebssei-1-o12",
+              "type": "choice",
+              "label": "Academic/Research Organization"
+            },
+            {
+              "name": "ebssei-1-o13",
+              "type": "choice",
+              "label": "Other",
+              "children": [
+                {
+                  "id": "ebssei-1-o13-text",
+                  "type": "textarea"
+                }
+              ]
+            }
+          ]
+        }
+      },
+      {
+        "id": "ebssei-1",
```

Comment:
Throws the following error because the id isn't unique:

![Screen Shot 2022-08-23 at 17 29 31](https://user-images.githubusercontent.com/27705928/186278146-d5b84bfe-2c73-4736-b9c0-ff33c0998425.png)

```suggestion
        "id": "ebssei-2",
```